### ISSUE TITLE
Create new UI with Auction and Bidding !

### ISSUE THREAD

**Issue by maektwain** [Cynsar-Foundation/ans-app#3]
**Is your feature request related to a problem? Please describe.**
ToP Level Domain names are coming , so we need to add the function where users can search and place a bid on the names for next 30 DAYS.

**Describe the solution you'd like**
A clear UI that works on top of the existing UI 

**Describe alternatives you've considered**

**Additional context**

**Comment by Devilla** [comment]
Yeah, Mockups wiil be required here for Create Auction, List Auction Page and Bidding Page respectively. So, one can start auction for particular TLD (say owner) by creating auction and then listing on the list Auctions Page. Then, the bidders can actually select the Auction for particular TLD from there and start bidding on it using a Bidding Page.
